### DISCUSSION TITLE
Adding domains for suppression

### DISCUSSION THREAD

**Discussion by Simboss15** [postalserver/postal#3153] [Help with using Postal]
Hello. Tell me if there is a possibility to add a domain in suppression so that it automatically receives the "held" status.

Example: *@mail.ru (all mail from the domain mail.ru ) get the "held" status

**Comment by willpower232** [comment]
Postal will just try and send what it is given, messages are only held because Postal believes that they genuinely cannot be sent.

If you're trying to block emails to a specific domain, it would be done better in whatever you're using to send messages with Postal.
